django_ipam/tests/base/test_api.py

```diff
@@ -63,3 +72,108 @@ def test_unvailable_request_api(self):
                                     content_type='application/json')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data, None)
+
+    def test_create_subnet_api(self):
+        data = {
+            'subnet': '10.0.0.0/32',
+            'description': 'Test Subnet'
+        }
+        response = self.client.post(reverse('ipam:subnet_list_create'),
+                                    data=json.dumps(data),
+                                    content_type='application/json')
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(str(Subnet.objects.first().subnet), '10.0.0.0/32')
+
+    def test_read_subnet_api(self):
+        subnet_id = self._create_subnet(subnet="fdb6:21b:a477::9f7/64").id
+        response = self.client.get(reverse('ipam:subnet', args=(subnet_id,)))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, str(subnet_id))
```

Comment:
this is not good enough, you should check one of the attributes of the JSON response in `response.data`.